org.moneymanagerex.MMEX.metainfo.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.moneymanagerex.MMEX</id>
+  
+  <name>Money Manager Ex (MMEX)</name>
+  <summary>Free, easy-to-use, personal finance software</summary>
+  
+  <metadata_license>MIT</metadata_license>
```

Comment:
`CC0-1.0` is prefered (MIT doesn't really apply to data)

Reply:
https://github.com/moneymanagerex/moneymanagerex/blob/master/resources/mmex.desktop